dht.go

```diff
@@ -3,10 +3,9 @@ package p2pd
 import (
 	"context"
 
-	pb "github.com/libp2p/go-libp2p-daemon/pb"
-
 	cid "github.com/ipfs/go-cid"
 	crypto "github.com/libp2p/go-libp2p-crypto"
+	pb "github.com/libp2p/go-libp2p-daemon/pb"
```

Comment:
this is a same package import, we keep those on their own. this doesn't get rewritten by gx.

also, a general advice -- there shouldn't be any changes in the diff to code you are not touching!

Reply:
Good to know. Replaced